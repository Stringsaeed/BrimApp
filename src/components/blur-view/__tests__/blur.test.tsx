import { screen } from "@testing-library/react-native";
import React from "react";

import render from "utils/test";

import BlurView from "../blur";

describe("BlurView", () => {
  it("should render correctly", () => {
    render(<BlurView />);
    expect(screen.toJSON()).toMatchSnapshot();
  });
});
