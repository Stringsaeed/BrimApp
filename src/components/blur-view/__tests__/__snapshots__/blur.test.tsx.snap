// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlurView should render correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
      },
      undefined,
    ]
  }
>
  <ViewManagerAdapter_ExpoBlurView
    blurReductionFactor={4}
    experimentalBlurMethod="none"
    intensity={50}
    style={
      {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
    tint="default"
  />
</View>
`;
